bundles/org.openhab.binding.bluetooth.dbusbluez/NOTICE

```diff
@@ -0,0 +1,46 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab-addons
+
+== Third-party Content
+
+TinyB Version: 0.5.1 
+* License:  MIT License
+* Project: https://github.com/intel-iot-devkit/tinyb
```

Comment:
You've included the wrong project information. You don't use tinyb in this binding.

Reply:
Fixed.